labs/lab4-soluce.md

```diff
@@ -1,3 +1,259 @@
 # Soluces for Lab 4
 
-TODO
+## Prerequisite:
+- Adding a Machine Learning instance to your deplyment
+- This instance should be at least 16 GB RAM and 8 vCPU
```

Comment:
I'm not sure we can do that with the trial accounts. Could you check this by creating a fake personal trial account?

Reply:
Ok I will test that tmw :)

Reply:
Nope we cannot add 16 GB RAM and 8 vCPU with a trial account.
The maximum is 2 GB RAM | 1 vCPU up to 8 vCPU.

Reply:
And can we play with infer with those nodes?

Reply:
We can show the demo on our laptops ate least, WDYT?

Reply:
Of course we can